tests/Kernel/CorporateBlocks/CorporateBlocksFooterTest.php

```diff
@@ -5,316 +5,264 @@
+    $actual = $section->filter('ul li:nth-child(2) > a');
+    $this->assertEquals($test_data['service_navigation'][1]['href'], $actual->attr('href'));
+    $this->assertEquals($test_data['service_navigation'][1]['label'], $actual->text());
 
-    $actual = $third_column->filter('ul li:nth-child(1) > a');
-    $this->assertEquals($test_data['about_eu_links'][0]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['about_eu_links'][0]['label'] . ' ', trim($actual->text()));
+    $section = $crawler->filter('footer.ecl-footer-core section.ecl-footer-core__section4');
 
-    $actual = $third_column->filter('ul li:nth-child(2) > a');
-    $this->assertEquals($test_data['about_eu_links'][1]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['about_eu_links'][1]['label'] . ' ', trim($actual->text()));
+    $actual = $section->filter('ul li:nth-child(1) > a');
+    $this->assertEquals($test_data['legal_navigation'][0]['href'], $actual->attr('href'));
+    $this->assertEquals($test_data['legal_navigation'][0]['label'], $actual->text());
 
-    $third_column = $crawler->filter('footer.ecl-footer div.ecl-footer__common div.ecl-footer__common-container');
-
-    $actual = $third_column->filter('a:nth-child(1)');
-    $this->assertEquals($test_data['bottom_links'][0]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['bottom_links'][0]['label'], trim($actual->text()));
-
-    $actual = $third_column->filter('a:nth-child(2)');
-    $this->assertEquals($test_data['bottom_links'][1]['href'], $actual->attr('href'));
-    $this->assertEquals($test_data['bottom_links'][1]['label'], trim($actual->text()));
-
-    $actual = $crawler->filter('footer.ecl-footer div.ecl-footer__common div.ecl-footer__common-container a.ecl-footer__common-link');
-    $this->assertCount(2, $actual);
-
-    /** @var \Drupal\Core\Config\ConfigFactoryInterface $config_factory */
-    $config_factory = \Drupal::service('config.factory');
-    $custom_footer_data = $this->getTestCustomFooterConfigsData();
-    foreach ($custom_footer_data as $config_name => $config_data) {
-      $config_factory->getEditable($config_name)->setData($config_data)->save();
-    }
-    \Drupal::configFactory()->getEditable('system.site')->set('name', 'Site Identity')->save();
-    $render = $this->buildBlock('oe_corporate_blocks_ec_footer', $config);
+    $actual = $section->filter('ul li:nth-child(2) > a');
+    $this->assertEquals($test_data['legal_navigation'][1]['href'], $actual->attr('href'));
+    $this->assertEquals($test_data['legal_navigation'][1]['label'], $actual->text());
+  }
 
-    $html = (string) $this->container->get('renderer')->renderRoot($render);
+  /**
+   * Test European Commission footer standardised block rendering.
+   */
+  public function testEcFooterStandardisedBlockRendering(): void {
+    $this->configFactory->getEditable('oe_theme.settings')->set('branding', 'standardised')->save();
+    $test_data = [];
```

Comment:
Why not use the test_data on the assertions? In similar way, there is a lot of repeated code blocks in this tests.